langcodes/__init__.py

```diff
@@ -429,138 +463,76 @@ def fill_likely_values(self) -> 'LanguageData':
         information on the Internet. (This is why the overall default is English,
         not Chinese.)
 
-        >>> str(LanguageData.get('zh-Hant').fill_likely_values())
+        >>> str(Language.get('zh-Hant').maximize())
         'zh-Hant-TW'
-        >>> str(LanguageData.get('zh-TW').fill_likely_values())
+        >>> str(Language.get('zh-TW').maximize())
         'zh-Hant-TW'
-        >>> str(LanguageData.get('ja').fill_likely_values())
+        >>> str(Language.get('ja').maximize())
         'ja-Jpan-JP'
-        >>> str(LanguageData.get('pt').fill_likely_values())
+        >>> str(Language.get('pt').maximize())
         'pt-Latn-BR'
-        >>> str(LanguageData.get('und-Arab').fill_likely_values())
+        >>> str(Language.get('und-Arab').maximize())
         'ar-Arab-EG'
-        >>> str(LanguageData.get('und-CH').fill_likely_values())
+        >>> str(Language.get('und-CH').maximize())
         'de-Latn-CH'
-        >>> str(LanguageData().fill_likely_values())    # 'MURICA.
+        >>> str(Language.make().maximize())    # 'MURICA.
         'en-Latn-US'
-        >>> str(LanguageData.get('und-ibe').fill_likely_values())
+        >>> str(Language.get('und-ibe').maximize())
         'en-ibe-Latn-US'
         """
+        if self._filled is not None:
+            return self._filled
+
         for broader in self.broaden():
-            tag = str(broader)
+            tag = broader.to_tag()
             if tag in DB.likely_subtags:
-                result = LanguageData.get(DB.likely_subtags[tag])
-                return result.update(self)
+                result = Language.get(DB.likely_subtags[tag], normalize=False)
+                result = result.update(self)
+                self._filled = result
+                return result
 
         raise RuntimeError(
             "Couldn't fill in likely values. This represents a problem with "
             "DB.likely_subtags."
         )
+    fill_likely_values = maximize
```

Comment:
The statements at 880 are at the module level, while this is a method of a class. I'll add a comment.